.gitbook/1-users/3-data-exchange/1-consume-data.md

```diff
@@ -0,0 +1,93 @@
+
+For more details about data deals, please see the [Data Deal](../../3-protocol-devs/1-dep-specs/2-data-deal.md) specification.
+
+
+## Query deals
+
+You can query a deal with its deal ID.
+```bash
+panacead query datadeal deal ${deal-id} \
+  --node ${chain-node-rpc-addr}
+```
+Also, you can query all deals registered in the chain.
+```bash
+panacead query datadeal deals \
+  --node ${chain-node-rpc-addr}
+```
+
+
+## Query consents
+
+If some data consumers have data which fits requirements of your data deal, they will submit consents to the chain.
+The consent means that the data provider agreed to provide their data to the specific data consumer.
+Also, each consent can contain a data validation certificate issued by an oracle, so that data consumers can trust the validity of data.
+
+As soon as data providers submit their consents, you can query all consents submitted to a specific data deal.
+```bash
+panacead query datadeal consents ${deal-id} \
+  --node ${chain-node-rpc-addr}
+```
+Or, you can query a specific consent which contains a certain data hash.
+```bash
+panacead query datadeal consent ${deal-id} ${data-hash} \
+  --node ${chain-node-rpc-addr}
+```
+
+For more details about data consents, please see the [Data Provider Consent](../../3-protocol-devs/1-dep-specs/3-data-provider-consent.md) specification.
+
+
+## Access data
+
+A data validation certificate issued by an oracle contains how the data can be accessed.
+For example, if the oracle decides to transmit data via [IPFS](https://ipfs.tech/), the certificate will contain a [CID](https://docs.ipfs.io/concepts/content-addressing/) of data.
+If so, you can access any IPFS node which is connected to the public IPFS network, and can obtain the data.
+
+For more details about data validation certificates, please see the [Data Validation](../../3-protocol-devs/1-dep-specs/4-data-validation.md) specification.
+
+In general, the data transmitted is encrypted by oracles, so that only the specific data consumer is able to decrypt it.
+Using the following REST API, you can get a secret key of each data from the oracle which issued the data validation certificate.
+```bash
+curl -v -X GET -H "Authorization: Bearer ${jwt}" \
+  "${oracle-url}/v0/data-deal/secret-key?deal-id=${deal-id}&data-hash=${data-hash}"
+```
+You must specify a JWT issued by yourself in order to prove that you are the data consumer who created the data deal.
+For that authentication, the JWT must be signed by your (data consumer's) chain account private key.
+
+We highly recommend to set the expiration of JWT as short as possible for security reasons.
+In near future, the protocol will adopt the 'nonce' concept to improve the security of authentications.
+
+Please note that the returned secret key is also encrypted, so that also the specific data consumer decrypt the key using his/her chain account private key.
```

Comment:
```suggestion
Please note that the returned secret key is also encrypted, so that only the specific data consumers can decrypt the key using his/her chain account private key.
```